docs/reference/technology/connectivity/cellular.md

```diff
@@ -0,0 +1,159 @@
+
+<span class="images">![](https://s3-us-west-2.amazonaws.com/mbed-os-docs-images/api-cellular-quick-start.png)<span>Figure 2. Connect to cellular network and open a socket</span></span>
+
+If you want to see code, you can go to our [cellular example](https://github.com/ARMmbed/mbed-os-example-cellular) or 
+our [advanced cellular example](https://github.com/ARMmbed/mbed-os-example-cellular-advanced).
+
+### Cellular connection management
+
+Applications can use cellular APIs to fine-control the cellular connection. Cellular APIs are structured based on main functionalities:
+
+- `CellularNetwork` for cellular network features, such as preferred operator and APN.
+- `CellularPower` for cellular module power control, such as enabling power save.
+- `CellularInformation` to read the cellular module type and firmware version.
+- `CellularSIM` to enter the PIN code and other SIM management functions.
+- `CellularSMS` to read and write SMS messages.
+
+The CellularDevice class encloses cellular APIs. Therefore, to use any cellular API, you need to get CellularDevice first. You can then use CellularDevice to open and close cellular APIs, as Figure 3 illustrates.
+
+<span class="images">![](https://s3-us-west-2.amazonaws.com/mbed-os-docs-images/api-cellular-device.png)<span>Figure 3. Use CellularDevice to open Cellular APIs</span></span>
+
+When an application has opened a cellular API, you can use it to request API methods. For example:
+
+    CellularNetwork *network  = cellularDevice->get_network();
+    if (network) {
+        printf("Local IP address is %s", network->get_ip_address());
+    }
+
+### UDP and TCP sockets
+
+If you want to use UDP or TCP sockets, you need an IP stack. Mbed OS cellular has an option to use either the LWIP stack, which is part of Mbed OS, or to use the IP stack on the cellular module. Figure 4 illustrates IP stack deployment.
+
+<span class="images">![](https://s3-us-west-2.amazonaws.com/mbed-os-docs-images/api-cellular-ip-stack.png)<span>Figure 4. IP stack can be used in PPP or AT mode</span></span>
+
+#### PPP mode with the LWIP stack on Mbed OS
+
+In PPP mode, the LWIP stack is linked as a part of the Mbed OS application. Sockets are implemented on the LWIP stack, and the modem is used over a Point-to-Point Protocol (PPP) link as a plain data pipe.
+
+You can enable PPP mode in the application configuration file:
+
+    "lwip.ppp-enabled": true
+
+#### AT mode with the IP stack on the modem
+
+In AT mode, the modem's internal IP stack is used over an AT link. Sockets are part of the IP stack, so AT commands are used to control sockets. Socket AT commands are modem specific, and you need to implement them on the Mbed OS side within a `CellularStack` class. You can browse `CellularStack` under the cellular/targets folder to learn how your cellular module supports AT sockets.
```

Comment:
Query: Who or what uses the modem's internal IP stack over an AT link? You, the user? Or something else? Also, who or what uses AT commands to control sockets?

Reply:
Above is a sentence "If you want to use UDP or TCP sockets, you need an IP stack."

This page is written to application developers so "you" always means an application developer, that is an API user.

I originally wrote this under APIs, but this was moved under Technology which may now cause some confusion... API (this page) and contributor (porting page) has identifiable readers... How about Technology??